src/inventory_client/inventory_client.go

```diff
@@ -250,15 +252,23 @@ func (c *inventoryClient) UploadIngressCa(ctx context.Context, ingressCA string,
 	return aserror.GetAssistedError(err)
 }
 
-func (c *inventoryClient) GetCluster(ctx context.Context) (*models.Cluster, error) {
-	cluster, err := c.ai.Installer.V2GetCluster(ctx, &installer.V2GetClusterParams{ClusterID: c.clusterId})
+func (c *inventoryClient) GetCluster(ctx context.Context, withHosts bool) (*models.Cluster, error) {
```

Comment:
I suggest to turn the boolean into a type, otherwise it's very hard to understand the code without going into the declaration of the function every time